cmd/image/upload/upload.go

```diff
@@ -75,7 +78,8 @@ func NewCmd() *cobra.Command {
 	cmd.Example = helpExample
 	cmdutil.SilenceUsage(cmd)
 
-	cmd.Flags().StringVarP(&config.KubeConfig, constants.FlagKubeconfig, constants.FlagKubeconfigShort, "", constants.FlagKubeconfigHelp)
+	cmd.Flags().StringVarP(&uploadOptions.KubeConfigPath, constants.FlagKubeconfig, constants.FlagKubeconfigShort, "", constants.FlagKubeconfigHelp)
+	cmd.Flags().StringVarP(&clusterConfigPath, constants.FlagConfig, "", "", constants.FlagConfigHelp)
```

Comment:
As a future enhancement, it would be nice to have CLI arguments for this so that users don't require a full cluster config just to upload an image.  It's going to be impossible to read without flag groups, so we would need to find a way to group arguments.

What we did with `olcnectl` was look at the arguments provided and only emit relevant help.  For example, you could do `olcnectl module create --environment myenv --module kubernetes --help` and it would only show arguments related to that module.  We could do something similar like `ocne image upload --type olvm --help` and only print those flags.

Reply:
olvm actually needs enough fields that it would be a pain not to use a config file of some sort.  With that said, i will make a jira for this enhancement